config/routes.rb

```diff
@@ -2,6 +2,7 @@
   defaults format: :json do
     post "/auth", to: "auth#create"
     get "/auth", to: "auth#show"
+    resources :users, only: [:show]
```

Comment:
Is the idea to just add routes one at a time? Since we're pulling from FB, I imagine we don't do any create/delete, but do we need a delete route to unlink your account?

Do we need an edit route for things like user bio?